tests/test_raw_content_spider.py

```diff
@@ -50,18 +34,21 @@ def test_download_recipe_contents_with_a_simple_response(self):
             request=http.Request('https://www.foo.com'),
             body='<html></html>')
 
-        self.mock_settings.get.return_value = '/foo/download/root'
         self.get_image_patch.return_value = 'test_image.jpg'
         spider = raw_content_spider.RawContentSpider()
+        spider._filepath_prefix = '/foo/download/root/20170102/030405Z'
```

Comment:
I kind of covered this in the high level note, but I feel like this part was problematic because we're mocking out actual interesting behavior of the system under test. Calculating the path is pretty simple, but it's not *so* trivial that we should drop tests lightly.

It's also the test knowing too much about `RawContentSpider`'s internals. From the test's perspective, `RawContentSpider` should have to call `datetime.utcnow` at some point, so it's kind of cheating by setting one of the instance's internal variables to prevent that from happening.